test/transmuxer.test.js

```diff
@@ -3170,7 +3170,7 @@ QUnit.test('ensures baseMediaDecodeTime for audio is not negative', function() {
     events.push(event);
   });
   audioSegmentStream.track.timelineStartInfo.baseMediaDecodeTime = 10;
-  audioSegmentStream.setEarliestDts(111);
+  audioSegmentStream.setEarliestDts(101);
```

Comment:
since baseMediaDecodeTime isn't used and this function is just a dumb setter now, we have to subtract ten so the tests stays mostly the same.